operators/pkg/tenant-controller/nextcloud_utils.go

```diff
@@ -32,6 +32,7 @@ var ncHeaders = map[string]string{"OCS-APIRequest": "true"}
 // GetUser gets the user with the corresponding username in nextcloud. It returns info about the existence of the user, the displayname of the user and if there are any errors
 func (ncA *NcActor) GetUser(ncUsername string) (found bool, displayname *string, err error) {
 	userEndpoint := ncA.buildOCSEndpoint(fmt.Sprintf("/users/%s", ncUsername))
+	ncA.Client.SetCookieJar(nil)
```

Comment:
I feel it may be enough to configure this once, when you initialize the client. WDYT?

Reply:
Yep you are right!

Reply:
I deployed and it works